dist/browser-warning.js

```diff
@@ -145,24 +145,75 @@
         $warningNoscript.parentNode.replaceChild($warningContainer, $warningNoscript);
 
         // set warning message
+        var $warningHeadline = document.getElementById('browser-warning-headline');
+        var $warningCopyLink = document.getElementById('browser-warning-copy-link');
         var $warningMessage = document.getElementById('browser-warning-message') || $warningContainer;
         if (warning === 'web-view') {
-            $warningMessage.textContent = 'You are viewing this page from inside another app. Please use a real browser.'
+            $warningHeadline.textContent = 'Please copy the link and paste it directly in Safari, Chrome or Firefox.';
+            $warningMessage.textContent = 'You\'re currently in a so-called in-app browser.';
+            $warningCopyLink.classList.add('show');
```

Comment:
Instead of having the button already in the markup and requiring css to hide/unhide it, you could just create the button here when required.